dttools/src/text_list.h

```diff
@@ -1,12 +1,18 @@
 #ifndef TEXT_LIST_H
 #define TEXT_LIST_H
 
+struct text_list {
```

Comment:
I think the idea here was to keep struct text_list private and access it through methods like `text_list_get`